modules/encrypt_seclib/src/Tests/EncryptEncryptDecryptTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+namespace Drupal\encrypt\Tests;
+
+/**
+ * Test basic encrypting and decripting of a string.
+ * 
+ * @group encrypt
+ */
+class EncryptEncryptDecryptTest extends \Drupal\simpletest\WebTestBase {
```

Comment:
Yea, I think we need to revisit testing as a while once we get everything in place.